use-case-guides/workday-request-time-off/README.md

```diff
@@ -99,10 +65,15 @@ We will be accessing Workday through the read-only WQL API and the read + write
 | Worker Data Query (WQL) | Get_References, Get_Workers, Get_Eligible_Time_Off_Plans, Get_Time_Off_Balance, Get_Positions | Permissions needed to retrieve worker information, including contact details, position, and time off balances. |
 | Time Off Request (ReST) | Put_Time_Off_Request, Get_Workers, Get_Positions | Permissions needed to submit time off requests and retrieve worker and position information. |
 
+![Domain%20Permissions](Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/Screenshot%202024-07-12%20at%2012.49.29 PM.png)
+
 
 > ℹ️ **You will be creating two ISUs, one in a staging environment, and one in production.**
 
 
+
+# For Creator Studio Developers
```

Comment:
- Input 1 - not sure why we need "input" for the API testing. It's not clear. Maybe it's a part of the API testing instructions

- API 2 - Should start with 1

- "P1: Sequential Action Chaining design pattern in your iPaaS to build this use case" there's no reference for this. Is it actually useful for developers to know this? Or is it a meaningless abstraction?

Once I get to the implementation section
1. I don't know how I would do this in CREST
2. Triggers, NL Slots, Inferred Slots, Actions, Caveats -- not sure how to apply these
3. Misunderstanding of query-triggered paths